src/Konami.tsx

```diff
@@ -1,8 +1,60 @@
 import React from 'react';
-import PropTypes from 'prop-types';
```

Comment:
You removed PropTypes. What if I want to use this lib in a non-TS project? How would the user know that they messed up the typing of some prop? If this was internal project it wouldn't be necessary, but this change cripples the usage of the library for plain JS projects.

If you want to migrate the package to TS make sure PropTypes is still supported for JS projects.

Reply:
I didn't know why to use PropTypes with TypeScript, but this is a good reason to use both.

I will undo the deletion of the PropTypes declaration so.

Reply:
For internal projects (i.e. your code is not being used by any third party) TS and PropTypes are "redundant". Well, not really redundant since TS check is made during compilation and PropTypes at runtime, so if you're checking for types TS is preferred but PropTypes can still be used to check for specific values (for example if a prop must receive a string in an enum).

However, this is a library not some internal project. Because of that it's expected that people will use this both with TS and JS, so by removing PropTypes the JS users will have zero checks in place.

I read some articles about TS and PropTypes, seems like it's possible and there are a few solutions for this. Good luck :)

Reply:
thank you, now that makes sense to me. It's done :)